tests/Import-DbaCsv.Tests.ps1

```diff
@@ -22,6 +22,7 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
     $CommaSeparatedWithHeader = "$script:appveyorlabrepo\csv\CommaSeparatedWithHeader.csv"
     $col1 = "$script:appveyorlabrepo\csv\cols.csv"
     $col2 = "$script:appveyorlabrepo\csv\col2.csv"
+    $pipe3 = "$script:appveyorlabrepo\csv\pipe3.psv"
```

Comment:
This file would have to be added with example data in order for it to be usable.

Reply:
The file needs to be added to the [appveyor-lab/csv](https://github.com/sqlcollaborative/appveyor-lab/tree/master/csv) folder.

Reply:
sweet, @MarkIannucci submitted a PR and I accepted